test/test_dataset.jl

```diff
@@ -50,8 +50,7 @@ const AG = ArchGDAL;
         AG.getlayer(dataset1, 0) do layer1
             @test AG.nfeature(layer1) == 4
         end
-        # the following test is to cover an example of `macro cplerr(code, message)`
```

Comment:
Is there still a test that covers the `cplerr` macro now?

Reply:
Unfortunately there isn't. That's why the unit test coverage has gone down by a bit. I haven't been able to find an example of a scenario where we get `CE_Failure` without GDAL.jl throwing a `GDAL.GDALError`. Happy to add such a test in if someone discovers it.

Reply:
You can use the setproj! function:
```julia
        @test_throws ErrorException AG.setproj!(dataset1, "nonsensestring")
```

Reply:
Thank you! https://github.com/yeesian/ArchGDAL.jl/pull/236/commits/330a3c951d690b9a9a8f7e4317759210f8f54f4e